example-registry/example.json

```
@@ -0,0 +1,7 @@
+{
+    "status": "registered",
+    "name": "example",
+    "specification": "https://w3c-ccg.github.io/did-spec/",
+    "registry": "DID Specification",
+    "contact": "W3C Credentials Community Group"
```

```suggestion
    "verifiableDataRegistry": "DID Specification",
    "contactName": "W3C Credentials Community Group",
    "contactEmail": "public-credentials@w3.org",
    "contactWebsite": "https://w3c-ccg.github.io/"
```
Please align these two fields with the latest: https://github.com/w3c/did-spec-registries/pull/360#pullrequestreview-799557934

---

I corrected this.